### PR TITLE
Components code refactoring

### LINKED ISSUE #20: Refactor Dashboard component into smaller modular components
Hi, congratulations for this project!

I'm thinking about solving opened issues, so I was reading this project code and I realized most of the application logic and rendering is concentrated inside Dashboard component.

Since there are plans to grow this project with other features listed in opened issues (filters, etc), do you think it could benefit from a refactor, for example, separating `<Dashboard>` into smaller components, like `<Developers>`, `<Analyses>` and `<Repositories>`? Of course there is some cost associated with this refactor, since some logic would have to be rewritten and then some data would have to be grouped together in the parent component (in this case, `<Dashboard>`... So that's why I'm questioning: do you think it is worth to refactor? :)

Thank you for your time!

### PR DESCRIPTION
**Pull request by edgarberlinck**
## Closes #20

### CODE REVIEW

**Comment by edgarberlinck** [comment]
@rafaveira3 Veja se está OK por favor.

**Comment by edgarberlinck** [comment]
@rafaveira3 Agora acho que está ok.

**Comment by edgarberlinck** [comment]
Sure. I’ll do it for us.

Em qua, 9 de out de 2019 às 11:16, Rafael dos Santos <
notifications@github.com> escreveu:

**Comment by edgarberlinck** [comment]
Done.

File: src/components/Graph.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { Paper } from '@material-ui/core'
+import { Bar, Doughnut, Pie } from 'react-chartjs-2'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const chartOptions = {
```
Comment (rafaveira3) [inline src/components/Graph.js]: These options may only be applicable to `Bar` components. Would you mind creating a new const that stores both `Pie` and `Doughnut` as well? 🙃

Comment (edgarberlinck) [reply src/components/Graph.js]: Done. I've created just a simple deafult options, but you can change it using props.

**Comment by edgarberlinck** [comment]
@rafaveira3 could you help us with issue #14 ?

**Comment by rafaveira3** [comment]
> @rafaveira3 could you help us with issue #14?

We are currently redesigning the dashboard and we will probably close that issue soon. Thanks for trying to help though.

**Pull request merged**